SECURITY.md

```diff
@@ -0,0 +1,25 @@
+Thanks for helping make Jane safe for everyone.
+
+## Security
+
+Jane takes the security of our software products and services seriously, including all of the open source code repositories managed through the [Jane](https://github.com/janeapp) organization.
+
+## Reporting Security Issues
+
+If you believe you have found a security vulnerability in any Jane-owned repository, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public Jane issues, discussions, or pull requests.**
```

Comment:
This should be self-explanatory, but do we want to explicitly call out spaces such as "social media" or "the Jane Facebook community" here?

Reply:
It's a good point. But I'm also wondering if it's necessary due to the context of anyone reading is on Jane's github organization 🤔 I could go either way on it.